cmd/minishift/cmd/timezone.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/minishift/timezone"
+	"time"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/minishift/minishift/cmd/minishift/cmd/util"
+	"github.com/minishift/minishift/cmd/minishift/state"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/cobra"
+)
+
+// stopCmd represents the stop command
+var (
+	timezoneCmd = &cobra.Command{
+		Use:   "timezone",
+		Short: "Set timezone for running Minishift Instance",
+		Long:  `Set timezone for running Minishift Instance, provided timezone should be valid.`,
+		Run:   runTimezone,
+	}
+	set string
+)
+
+func runTimezone(cmd *cobra.Command, args []string) {
+	api := libmachine.NewClient(state.InstanceDirs.Home, state.InstanceDirs.Certs)
+	defer api.Close()
+
+	// if VM does not exist, exit with error
+	util.ExitIfUndefined(api, constants.MachineName)
+
+	hostVm, err := api.Load(constants.MachineName)
+	if err != nil {
+		atexit.ExitWithMessage(1, err.Error())
+	}
+
+	if set == "" {
```

Comment:
This gives an error on a bare `m timezone` command. This would not be expected

Reply:
please make this hidden for now, as i am not convinced `timezone` is the right place.

Reply:
@gbraad @praveenkumar I think timezone stuff should be separate command as we should not club with anything else.

Reply:
It is related to general VM/instance setup

creating a new top-level command is in my opinion too much credit to such a small functionality. It should be grouped inside a general Vm/Instance setup command. Imagine I want to modify the nameservers inside the VM, do we create a new top-level command `dns` (exists) for this, or `nameservers` or just `network`. what if I want to setup commands that start at boot time? `minishift boot` ? I think this is all `minishift machine`  related